### PR TITLE
Correct Azure Key Vault Certificates .md files content inconsistency and unclear

### PR DESCRIPTION
**Pull request by FredGao-new**
Create PR to address this [issue](https://github.com/Azure/azure-sdk-for-java/issues/10057).

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Azure Key Vault Certificate client library for Java
-Azure Key Vault allows you to create and store certificates in the Key Vault. Azure Key Vault client supports certificates backed by Rsa keys and Ec keys. It allows you to securely manage, tightly control your certificates.
+Azure Key Vault allows you to create and store certificates in the Key Vault. Azure Key Vault Certificate client Library supports certificates backed by Rsa keys and Ec keys. It allows you to securely manage, tightly control your certificates.
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:2]: RSA

EC

all caps.

Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:2]: Here's how I would rewrite this: 

Azure Key Vault allows you to securely manage and tightly control your certificates.  Azure Key Vault Certificate client library supports certificates backed by RSA and EC keys.

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
  Multiple certificates, and multiple versions of the same certificate, can be kept in the Key Vault. Cryptographic keys in Key Vault backing the certificates are represented as [JSON Web Key [JWK]](https://tools.ietf.org/html/rfc7517) objects. This library offers operations to create, retrieve, update, delete, purge, backup, restore and list the certificates and its versions.
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:4]: restore, 

don't forget the oxford comma

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 3]
```diff
@@ -31,12 +31,12 @@ Maven dependency for Azure Key Client library. Add it to your project's pom file
     ```
 
 ### Authenticate the client
-In order to interact with the Key Vault service, you'll need to create an instance of the [CertificateClient](#create-certificate-client) class. You would need a **vault url** and **client secret credentials (client id, client key, tenant id)** to instantiate a client object using the default `AzureCredential` examples shown in this document.
+In order to interact with the Key Vault service, you'll need to create an instance of the [CertificateClient](#create-certificate-client) class. You would need a **vault url** and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object using the `DefaultAzureCredential` examples shown in this document.
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:34]: remove "would"

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 4]
```diff
@@ -58,7 +58,7 @@ Here is [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to
     }
     ```
 
-* Use the above returned credentials information to set **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) environment variables. The following example shows a way to do this in Bash:
+* Use the above returned credentials information to set **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password) and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Bash:
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:61]: comma before and..... ", and"

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 5]
```diff
@@ -69,7 +69,7 @@ Here is [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to
 * Grant the above mentioned application authorization to perform key operations on the keyvault:
```
Comment (samvaity) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:69]: to perform **certificate** operations?

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 6]
```diff
@@ -117,15 +118,19 @@ The following sections provide several code snippets covering some of the most c
 
 ### Create a Certificate
 
-Create a Certificate to be stored in the Azure Key Vault.
-- `beginCreateCertificate` creates a new certificate in the key vault. if the certificate with name already exists then a new version of the certificate is created.
+Create a certificate to be stored in the Azure Key Vault.
+- `beginCreateCertificate` creates a new certificate in the Azure Key Vault. if the certificate with name already exists then a new version of the certificate is created.
```
Comment (samvaity) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:122]: ~~. if~~ -->  . If

Comment (samvaity) [reply sdk/keyvault/azure-security-keyvault-certificates/README.md:122]: Should all certificate be updated to Certificate?

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 7]
```diff
@@ -271,7 +274,7 @@ List the certificates in the key vault by calling `listPropertiesOfCertificates`
 ```Java
 // The List Certificates operation returns certificates without their full properties, so for each certificate returned we call `getCertificate` to get all its attributes excluding the policy.
 certificateAsyncClient.listPropertiesOfCertificates()
-    .subscribe(certificateProperties -> certificateAsyncClient.getCertificate(certificateProperties.getName(),
+    .subscribe(certificateProperties -> certificateAsyncClient.getCertificateVersion(certificateProperties.getName(),
```
Comment (samvaity) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:277]: [Line](https://github.com/Azure/azure-sdk-for-java/pull/10118/files#diff-824e08966bb186e6f79351c1a042d490R272) update to `Key Vault`.

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 8]
```diff
@@ -31,12 +31,12 @@ Maven dependency for Azure Key Client library. Add it to your project's pom file
     ```
 
 ### Authenticate the client
-In order to interact with the Key Vault service, you'll need to create an instance of the [CertificateClient](#create-certificate-client) class. You would need a **vault url** and **client secret credentials (client id, client key, tenant id)** to instantiate a client object using the default `AzureCredential` examples shown in this document.
+In order to interact with the Key Vault service, you'll need to create an instance of the [CertificateClient](#create-certificate-client) class. You need a **vault url** and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object using the `DefaultAzureCredential` examples shown in this document.
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:34]:
```suggestion
In order to interact with the Azure Key Vault service, you'll need to create an instance of the [CertificateClient](#create-certificate-client) class. You need a **vault url** and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object using the `DefaultAzureCredential` examples shown in this document.
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 9]
```diff
@@ -117,15 +118,19 @@ The following sections provide several code snippets covering some of the most c
...
-import com.azure.security.keyvault.certificates.models.CertificateOperation;
 import com.azure.security.keyvault.certificates.CertificateClient;
+import com.azure.security.keyvault.certificates.CertificateClientBuilder;
+import com.azure.security.keyvault.certificates.models.CertificateOperation;
+import com.azure.security.keyvault.certificates.models.CertificatePolicy;
+import com.azure.security.keyvault.certificates.models.KeyVaultCertificate;
+import com.azure.security.keyvault.certificates.models.KeyVaultCertificateWithPolicy; 
 
 CertificateClient certificateClient = new CertificateClientBuilder()
         .vaultUrl(<your-vault-url>)
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:136]:
```suggestion
        .vaultUrl(<your-key-vault-url>)
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 10]
```diff
@@ -192,7 +197,7 @@ for (CertificateProperties certificateProperties : certificateClient.listPropert
 ```
 
 ### Async API
-The following sections provide several code snippets covering some of the most common asynchronous Azure Key Vault Key Service tasks, including:
+The following sections provide several code snippets covering some of the most common asynchronous Azure Key Vault Certificate Service tasks, including:
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:200]:
```suggestion
The following sections provide several code snippets covering some of the most common asynchronous Azure Key Vault Certificate service tasks, including:
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 11]
```diff
@@ -192,7 +197,7 @@ for (CertificateProperties certificateProperties : certificateClient.listPropert
...
 - [Create a Certificate Asynchronously](#create-a-certificate-asynchronously)
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:201]:
```suggestion
- [Create a certificate asynchronously](#create-a-certificate-asynchronously)
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 12]
```diff
@@ -192,7 +197,7 @@ for (CertificateProperties certificateProperties : certificateClient.listPropert
...
 - [Retrieve a Certificate Asynchronously](#retrieve-a-certificate-asynchronously)
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:202]:
```suggestion
- [Retrieve a certificate asynchronously](#retrieve-a-certificate-asynchronously)
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 13]
```diff
@@ -192,7 +197,7 @@ for (CertificateProperties certificateProperties : certificateClient.listPropert
...
 - [Update an existing Certificate Asynchronously](#update-an-existing-certificate-asynchronously)
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:203]:
```suggestion
- [Update an existing certificate asynchronously](#update-an-existing-certificate-asynchronously)
```
and so on.

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 14]
```diff
@@ -203,15 +208,13 @@ The following sections provide several code snippets covering some of the most c
 
 ### Create a Certificate Asynchronously
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:209]:
```suggestion
### Create a certificate asynchronously
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 15]
```diff
@@ -203,15 +208,13 @@ The following sections provide several code snippets covering some of the most c
...
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:210]:
```suggestion
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 16]
```diff
@@ -203,15 +208,13 @@ The following sections provide several code snippets covering some of the most c
...
 
-Create a Certificate to be stored in the Azure Key Vault.
-- `beginCreateCertificate` creates a new certificate in the key vault. if the certificate with name already exists then a new version of the certificate is created.
+Create a certificate to be stored in the Azure Key Vault.
+- `beginCreateCertificate` creates a new certificate in the Azure Key Vault. if the certificate with name already exists then a new version of the certificate is created.
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:212]:
```suggestion
- `beginCreateCertificate` creates a new certificate in the Azure Key Vault. If a certificate with same name already exists then a new version of the certificate is created.
```

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 17]
```diff
@@ -203,15 +208,13 @@ The following sections provide several code snippets covering some of the most c
...
 ```Java
 import com.azure.identity.DefaultAzureCredentialBuilder;
-import com.azure.security.keyvault.certificates.models.Certificate;
-import com.azure.security.keyvault.certificates.models.CertificatePolicy;
-import com.azure.security.keyvault.certificates.models.CertificateOperation;
 import com.azure.security.keyvault.certificates.CertificateAsyncClient;
+import com.azure.security.keyvault.certificates.models.CertificatePolicy; 
 
 //Creates a certificate using the default policy and polls on its progress.
 certificateAsyncClient.beginCreateCertificate("certificateName", CertificatePolicy.getDefault())
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:220]:
```suggestion
certificateAsyncClient.beginCreateCertificate("<certificate-name>", CertificatePolicy.getDefault())
```
And all other similar instances.

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 18]
```diff
@@ -271,7 +274,7 @@ List the certificates in the key vault by calling `listPropertiesOfCertificates`
 ```Java
 // The List Certificates operation returns certificates without their full properties, so for each certificate returned we call `getCertificate` to get all its attributes excluding the policy.
 certificateAsyncClient.listPropertiesOfCertificates()
-    .subscribe(certificateProperties -> certificateAsyncClient.getCertificate(certificateProperties.getName(),
+    .subscribe(certificateProperties -> certificateAsyncClient.getCertificateVersion(certificateProperties.getName(),
      certificateProperties.getVersion());
         .subscribe(certificateResponse -> System.out.printf("Received certificate with name %s and key id %s",
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:279]:
```suggestion
        .subscribe(certificateResponse -> System.out.printf("Received certificate with name \"%s\" and key id %s",
```
And so on for all other similar instances.

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 19]
```diff
@@ -223,85 +219,79 @@ certificateAsyncClient.beginCreateCertificate("certificateName", CertificatePoli
...
+Delete an existing certificate by calling `beginDeleteCertificate`.
 
 ```java
-certificateAsyncClient.beginDeleteCertificate("certificateName")
+certificateAsyncClient.beginDeleteCertificate("<certificate-name>")
     .subscribe(pollResponse -> {
-        System.out.println("Delete Status: " + pollResponse.getStatus().toString());
-        System.out.println("Delete Certificate Name: " + pollResponse.getValue().getName());
-        System.out.println("Certificate Delete Date: " + pollResponse.getValue().getDeletedOn().toString());
+        System.out.printf("Deletetion status: %s\n", pollResponse.getStatus().toString());
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:254]: typo , should be Deletion.

Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:254]:
```suggestion
        System.out.printf("Deletion status: %s\n", pollResponse.getStatus().toString());
```

**Pull request merged**